docs/en/rules/Azure.AKS.PodIdentity.md

```diff
@@ -0,0 +1,64 @@
+---
+severity: Important
+pillar: Operational Excellence
+category: Configuration
+resource: Azure Kubernetes Service
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.AKS.PodIdentity/
+---
+
+# Use managed identities for AKS pod authentication
+
+## SYNOPSIS
+
+Configure AKS clusters to use AAD pod identities to access Azure resources securely.
+
+## DESCRIPTION
+
+AAD pod identities allows AKS clusters to assign a user identity to a pod in Kubernetes.
+
+Administrators create identities and bindings as Kubernetes primitives that allow pods to access
+Azure resources that rely on Azure AD as an identity provider.
+
+## RECOMMENDATION
+
+Consider enabling AAD pod identities on AKS clusters.
+
+## EXAMPLES
+
```

Comment:
Would be good to have a Bicep and Template example. Maybe just update the docs code and build to get both.

https://github.com/Azure/PSRule.Rules.Azure/blob/816c5c5086cd1c300f02781f9e8a952c9011bc10/docs/examples-aks.bicep